tests/integration/dual-channel-replication.tcl

```diff
@@ -871,6 +871,7 @@ start_server {tags {"dual-channel-replication external:skip"}} {
                 fail "Primary did not free repl buf block after sync failure"
             }
             wait_and_resume_process 0
+            $replica debug pause-after-fork 0
```

Comment:
Just to understand: this needs to be disabled before reading the log msgs?

Reply:
It can go anywhere after the wait_and_continue, but before the end of the test.

Reply:
If the replica is getting stuck in `sync` loops, we should disable it ASAP, right? Not after the three `stop_write_load` in the end of the test case.

We can disable `pause-after-fork` here, inside the test case (the `test { ... }` block), but then I think we should also move the enabling of `pause-after-fork` inside the test case, rather than before the start of the test case. Symmetry like that makes it easier to follow the logic, and also to skip test cases using `--only` and similar options.

Reply:
> If the replica is getting stuck in sync loops, we should disable it ASAP, right? Not after the three stop_write_load in the end of the test case.

The stoping of the load is actually outside of the test, I agree we should disable it before that part of the test. 

Addressed your other comment as well.